### DISCUSSION TITLE
Podcast episodes

### DISCUSSION THREAD

**Discussion by mdphillips1** [openaudible/openaudible#1061] [Q&A] [answered]
Hi there,

Just wondering if there is the option to access, download, convert etc. any podcast episodes in one's Audible library?  On OpenAudible 3.6.2, I have chosen the dropdown menu near the top, and clicked on 'Podcasts', but it doesn't seem to come up with anything, despite the fact that I have a lot of podcasts in my library!

I'm using OpenAudible on Linux Ubuntu 20.04, in case that's of any relevance.

Thanks
mdphillips1

**Comment by Cfomodz** [comment]
Podcast support was added in **version 4.2.0** (May 2024). Your version 3.6.2 doesn't have this feature.

### Steps to Fix

1. **Update OpenAudible**
   - Download the latest version: [openaudible.org](https://openaudible.org)
   - Linux: Use the [AppImage](https://openaudible.org/latest/OpenAudible_x86_64.AppImage)

2. **Enable Podcasts**
   - Open OpenAudible
   - Go to **Manage Libraries** 
   - Check **"Enable Podcasts"**
   - Click OK

3. **Sync Library**
   - Click **Full Library Sync** (not Quick Sync)
   - Podcasts will appear in the "Podcasts" filter

---

## Note

OpenAudible only downloads podcast episodes that are **saved to your Audible library**. Simply subscribing to a podcast isn't enough - you must add individual episodes to your library for them to sync.

**Comment by openaudible** [comment, marked as answer]
Try updating to the latest version.

Some earlier versions incorrectly disabled podcasts even if the Library: Podcasts checkbox was checked (which it needs to be.)

Select "Podcasts" in the Search bar at the upper left of the screen.

Podcasts should be downloadable.
